doc/sphinx/advanced_methods.rst

```diff
@@ -118,6 +118,87 @@ The following limitations currently apply for the collision detection:
 * The ``"bind at point of collision"`` approach cannot handle collisions
   between virtual sites
 
+.. _Deleting bonds when particles are pulled apart:
+
+Deleting bonds when particles are pulled apart
+----------------------------------------------
+
+With this feature, bonds between particles can be deleted automatically
+when the bond length exceeds a critical distance. This is used to model
+breakable bonds.
+
+The bond breakage action is specified for individual bonds via the system
+:attr:`~espressomd.system.System.bond_breakage` attribute.
+
+Several modes are available:
+
+* ``"revert_center_bond"``: delete a bond from the first particle
```

Comment:
I'm not happy with the names. The word center should not appear in the name for the methdo that just deletes the bond. `delete_bond` or similar should do. Please take into account that this might be used independently from the collision detection.  The revert_vs_bond action actually does not only that but also deletes bonds between the related physcal paricles. Maybe `revert_vs_non_sliding_contact` if we want something independent from collision detection, or otherwise the name from the core `rvert_bind_at_point_of_collision`.

Reply:
I agree with "delete bond".

For the latter I would fall back on the name of the collision detection. People can use this independently, but might not do so. And I think it is hard to imagine what it means when you are not using the collision detection.

Also another thought: What happens when a user connect vs manually without connecting the center particles...